model_compression_toolkit/exporter/model_wrapper/pytorch/builder/fully_quantized_model_builder.py

```diff
@@ -37,10 +40,29 @@ def fully_quantized_wrapper(node: common.BaseNode,
         Returns: Wrapped layer
 
         """
-        weight_quantizers, activation_quantizers = get_quantization_quantizers(node)
-        wrapped_layer = PytorchQuantizationWrapper(module, weight_quantizers, activation_quantizers)
-        return wrapped_layer
+        weight_quantizers, _ = get_quantization_quantizers(node)
+        if len(weight_quantizers) > 0:
+            return PytorchQuantizationWrapper(module, weight_quantizers)
+        return module
 
+    def get_activation_quantizer_holder(node: BaseNode) -> Union[None, Callable]:
```

Comment:
Fix type hints

Reply:
fix to what? it is Union[None, Callable]